src/main/scala/edu/twitter/model/impl/gradientboosting/GradientBoostingModel.scala

```diff
@@ -11,7 +11,7 @@ import org.apache.spark.mllib.tree.model.GradientBoostedTreesModel
   */
 class GradientBoostingModel(model: GradientBoostedTreesModel) extends GenericModel {
 
-  val name = "GradientBoosting"
+  val name: String = GradientBoostingModel.name
   val hashingTF = new HashingTF(2000)
```

Comment:
use private

Reply:
no actually we need this a public as it is implementing the `name` method in the generic model interface.

Reply:
my comment on hashingTF

Reply:
oh sorry, will fix it :+1: